client/src/pages/contentPages/Groups.js

```diff
@@ -1,7 +1,57 @@
 import React from 'react';
+import { Grid, Typography, Container } from '@material-ui/core';
+import { makeStyles } from '@material-ui/core/styles';
+import { groups } from '../../data/mockData.js'
+import Navbar from '../layout/Navbar';
+import GroupCard from '../../components/Group/GroupCard';
+
+const useStyles = makeStyles((theme) => ({
+
+    cardGrid: {
+      paddingTop: theme.spacing(8),
+      paddingBottom: theme.spacing(8),
+    }
+    
+  }));
 
 const Groups = () => {
-    return <div>Groups Page</div>;
-};
+    const classes = useStyles();
+
+    return (
+        <Grid>
+        <Navbar />
```

Comment:
Maybe on the next merge, we will move the Navbar to a higher-order component so it will be on top of every page that needs it.